aptos-move/aptos-vm/src/transaction_metadata.rs

```diff
@@ -66,20 +69,66 @@ impl TransactionMetadata {
                 // Deprecated. Return an empty vec because we cannot do anything
                 // else here, only `unreachable!` otherwise.
                 TransactionPayload::ModuleBundle(_) => vec![],
+
+                TransactionPayload::Payload(TransactionPayloadInner::V1 {
+                    executable,
+                    extra_config,
+                }) => {
+                    if extra_config.is_multisig() {
+                        vec![]
```

Comment:
I'm not sure if this is expected, before there's no script in multi-sig payload but now it has, having an empty hash seems wrong, or we need to have a clear flow to reject multi-sig script (api layer is not sufficient)

Reply:
To be honest, I'm not a fan of splitting multisig into executable and extra_config. @igor-aptos preferred this approach during our initial design. So, I followed it.
At the moment, validate_transaction in aptos_vm.rs discards transactions with multi-sig script. Is that fine?

Reply:
We currently have
```
TransactionExectuable {
   EntryFunction(e),
   Script(s),
   Emtpy
}
TransactionExtraConfig {
    multisig_address: Option<Address>,
    replay_protection_nonce: Option<u64>,
}
```

Alternative could be
```
TransactionExectuable {
   EntryFunction(e),
   Script(s),
   Multisig(m),
}
TransactionExtraConfig {
    replay_protection_nonce: Option<u64>,
}
```

Reply:
> can we return here value that would be if multisig supported scripts, instead of dummy one?

We can keep the current implementation and support multisig scripts as well. We can take @movekevin's feedback implement that in a separate PR.